modules/Bio/EnsEMBL/IO/Translator/VariationFeature.pm

```diff
@@ -31,17 +32,39 @@ use Carp;
 
 use base qw/Bio::EnsEMBL::IO::Translator::Feature/;
 
+sub seqname {
+  my $self = shift;
+  my $object = shift;
+  
+  return $object->seq_region_name;
+}
+
+sub start {
+  my $self = shift;
+  my $object = shift;
 
-=head2 get_itemRgb
+  return ( $object->seq_region_start() > $object->end() ) ? $object->end() : $object->seq_region_start();
```

Comment:
Oups I overlooked that: I was only interested in retrieving the seq_region_start in the tests I did